packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -86,9 +86,7 @@ const ContractInfo = ({
     const setHintMessage = () => {
         if (['VANILLALONGCALL', 'VANILLALONGPUT'].includes(type)) {
             return (
-                <Localize
-                    i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.'
-                />
+                <Localize i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.' />
```

Comment:
@adrienne-deriv same question as in https://github.com/binary-com/deriv-app/pull/9013/files#r1238618733 🙂

Reply:
These files are from conflict resolution, trying to get them resolved and are not my changes